<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transform_panel</class>
 <widget class="QWidget" name="transform_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>transform_panel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(245,245,245)</string>
  </property>
  <widget class="QLabel" name="label_transform_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_transform_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_setting">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="high_quantity">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cut_part">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_list">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="play_select">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_start">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_file_up">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>701</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="under_nameof_output_file">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_file_under">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="under_change">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>680</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>679</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>OUT_PUT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>输出至源文件目录</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清空列表</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>添加文件夹</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
